cloudpathlib/azure/azblobclient.py

```diff
@@ -1,9 +1,14 @@
 from datetime import datetime, timedelta
 import mimetypes
 import os
+from http import HTTPStatus
 from pathlib import Path
 from typing import Any, Callable, Dict, Iterable, Optional, Tuple, Union
 
+try:
+    from typing import cast
+except ImportError:
+    from typing_extensions import cast
```

Comment:
Wouldn't `from typing_extensions import cast` suffice, as it already does a very similar thing under the hood?